### PR TITLE
Add rust winapi dependencies.

### LINKED ISSUE #60: Linking libraries issue
Sorry i'm not very familiar with the auto{conf, ...} thing. But i hope someone who is more familiar with them can fix it. 

The problem itself is very clear:
Under windows the generated rust static library have 5 dependency libraries, three of them already linked but two not, namely  -lws2_32   -luserenv.  The cargo build can correctly displays this information on the screen. However the emacs linking doesn't have these flags. I think it needs to modified to either add these libraries on Windows platform or detect cargo output and do something.

### PR DESCRIPTION
**Pull request by crlf0710**
Resolves #60.

### CODE REVIEW

**Comment by crlf0710** [comment]
No, it didn't work. There's order problem. Let me try again.

**Comment by crlf0710** [comment]
This time the linking problem is gone.

**Comment by Wilfred** [comment]
Thanks.
